app/src/test/kotlin/fr/free/nrw/commons/delete/DeleteHelperTest.kt

```diff
@@ -65,9 +66,14 @@ class DeleteHelperTest {
         `when`(media?.displayTitle).thenReturn("Test file")
         media?.filename="Test file.jpg"
 
+        val creatorName = "Creator"
+        val noPageSuffix = " (page does not exist)"
```

Comment:
I assume this variable can be deleted now

Reply:
And related parts of test should be updated

Reply:
Thanks for your help @neslihanturan :)
 I'm new to tests, sorry. Why those variables should be deleted? I added them to make it clear  that creator name can consist of two parts: name itself and suffix "page does not exist" if user hasn't created page yet. And the suffix should be stripped from a string in order to post correct notification to creator, which I check with `verify(pageEditClient)?.appendEdit(eq("User_Talk:$creatorName"), ArgumentMatchers.anyString(), ArgumentMatchers.anyString())`

Reply:
`noPageSuffix` is only used in one place, we use variables to give a clearer meaning to some data or to reuse some data.
You can keep `creatorName`

Reply:
Okay, thanks